assume/common/grid_utils.py

```diff
@@ -27,16 +27,19 @@ def add_generators(
         index=network.snapshots,
         columns=generators.index,
     )
-
+    gen_c = generators.copy()
```

Comment:
this gen_c is created but never used? Shouldn't it be passed as **gen_c instread of **generators?